### PR TITLE
Fix views crashing when description or content fields are null

### PR DESCRIPTION
**Pull request by mmarkusik**
I want to merge this change because it fixes pages / product details crashing when content or description fields are null

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

File: src/views/Article/View.tsx [hunk 1]
```diff
@@ -13,8 +13,8 @@ import { TypedArticleQuery } from "./query";
 
 import "./scss/index.scss";
 
-const canDisplay = page =>
-  maybe(() => !!page && !!page.title && !!page.content);
+const canDisplay = page => maybe(() => !!page && !!page.title);
```
Comment (orzechdev) [inline src/views/Article/View.tsx:16]: maybe? 👮 
```suggestion
const canDisplay = page => !!page?.title;
```

File: src/views/Product/View.tsx [hunk 2]
```diff
@@ -19,13 +19,8 @@ import { IProps } from "./types";
...
 
 const canDisplay = (product: ProductDetails_product) =>
-  maybe(
-    () =>
-      !!product.description &&
-      !!product.name &&
-      !!product.pricing &&
-      !!product.variants
-  );
+  maybe(() => !!product.name && !!product.pricing && !!product.variants);
```
Comment (orzechdev) [inline src/views/Product/View.tsx:22]: 👮

Comment (dominik-zeglen) [reply src/views/Product/View.tsx:22]: ☝️

Comment (maarcingebala) [reply src/views/Product/View.tsx:22]: 👀

Comment (mmarkusik) [reply src/views/Product/View.tsx]: ✅

**Review by maarcingebala — APPROVED** [review]
Tested locally, it works 👍

**Review by orzechdev — APPROVED** [review]

**Pull request merged**